src/utils/creditCardType.js

```diff
@@ -0,0 +1,24 @@
+const CREDIT_CARDS = [
```

Comment:
I would create **creditCardTypes.js** in _constants_ directory, with cards defined as:
CREDIT_CARD_VISA = 'VISA'
CREDIT_CARD_UNKNOWN = 'UNKNOWN'
etc.